CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ All notable changes to this project will be documented in this file. The format
   - convert listing address jsonb columns to separate address tables
   - removed unused inverse relations from entities
   - recreated foreign keys constraints for `application_flagged_set_applications_applications`, `listings_leasing_agents_user_accounts`, `property_group_properties_property` and add missing `NOT NULL` migration for listing name column
+  - add google translations for GET listing API call
```

Comment:
Can you add the right formatting here? It should look like `- description (([#1590](https://github.com/bloom-housing/bloom/pull/1590)) (contributor name)`

Reply:
Sure and done.